packages/react-native-lockdown/src/makeGetPolyfills.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Generates a list of polyfills with SES to include in the Metro bundle. This
+ * sets up a Hardened JavaScript environment on React Native apps at earliest
+ * point of entry.
+ *
+ * @example Metro.config.js const makeGetPolyfills =
+ * require('@lavamoat/react-native-lockdown/makeGetPolyfills'); module.exports =
+ * { serializer: { getPolyfills: ({platform}) => makeGetPolyfills({engine:
+ * 'hermes', polyfills: ['reflect-metadata']}), }, };
```

Comment:
extract to README.md

Reply:
<s>@lavamoat/react-native-lockdown/makeGetPolyfills</s>

@lavamoat/react-native-lockdown

makeGetPolyfills is now the `main` export